src/styles/utils/_variables.scss

```
@@ -0,0 +1,12 @@
+$days-amount: 31;
+$gap: 1px;
+$item-length: 100px;
+
+$week-days:
+  "mon" 0,
+  "tue" #{$item-length + $gap},
+  "wed" #{$item-length * 2 + $gap * 2},
+  "thu" #{$item-length * 3 + $gap * 3},
+  "fri" #{$item-length * 4 + $gap * 4},
+  "sat" #{$item-length * 5 + $gap * 5},
+  "sun" #{$item-length * 6 + $gap * 6};
```

```suggestion
  "tue" #{$item-length + $gap},
  "wed" #{$item-length * 2 + $gap * 2},
  "thu" #{$item-length * 3 + $gap * 3},
  "fri" #{$item-length * 4 + $gap * 4},
  "sat" #{$item-length * 5 + $gap * 5},
  "sun" #{$item-length * 6 + $gap * 6};
```
move calculations inside loop